src/containers/transaction/Transaction.js

```diff
@@ -32,19 +32,15 @@ const generateTransfersArr = async transaction => {
           const amount = notification.state.value.find(
```

Comment:
I dont think this file should actually have any changes in it for this task... Try pulling the latest of `master` 

Reply:
Should be good now, let me know if I missed anything else!